submissions/Vlada-khom/html-css-popup/styles.css

```diff
@@ -0,0 +1,200 @@
+.background {
```

Comment:
No need to add `styles.css` if you already add `styles.scss`. Your PR should contain files for review, please exclude compiled files.